setup.py

```diff
@@ -73,7 +73,6 @@ def run(self):
         "Operating System :: Microsoft :: Windows",
         "Operating System :: POSIX",
         "Programming Language :: C++",
-        "Programming Language :: Python :: 2.7",
         "Programming Language :: Python :: 3",
```

Comment:
Considering that `f-strings` are being used, torchaudio requires `>=3.6`. Should we be more specific and list

```py
"Programming Language :: Python :: 3.6",
"Programming Language :: Python :: 3.7",
"Programming Language :: Python :: 3.8",

```

?

Reply:
Thanks for the suggestion. I think that makes sense.

(FYI: The reason we can use f-string is that PyTorch core does not support 3.5. The support for 3.5. was dropped in July 2020 release IIRC)